cmd/git-init/main.go

```diff
@@ -59,7 +58,7 @@ func main() {
 		logger.Fatalf("Error parsing revision %s of git repository: %s", fetchSpec.Revision, err)
 	}
 	resourceName := os.Getenv("TEKTON_RESOURCE_NAME")
-	output := []v1beta1.PipelineResourceResult{
+	output := []termination.PipelineTerminationResult{
```

Comment:
SidecarLogs also uses this to store results: https://github.com/tektoncd/pipeline/blob/a7e39c2e87a5bf41d8e5bd93858a6dfba7a1e31b/internal/sidecarlogresults/sidecarlogresults.go#L187-L191

I wonder if we should just call it `PipelineResult` (the extraction method should not be in the name I think)

Reply:
How about just `Result`? Is that taken?

Reply:
Thanks Chitrang! I think it makes sense to be put together with the above suggestions of moving it to the `pkg/internal`. Then we can reference it as either 'internal.result' or 'internal.pipelineresult'